cdap-app-fabric/src/test/java/co/cask/cdap/internal/AppFabricClient.java

```diff
@@ -398,7 +398,9 @@ public void deployApplication(Id.Application appId, AppRequest appRequest) throw
                                                                appId.getNamespaceId(), appId.getId());
     Preconditions.checkNotNull(bodyConsumer, "BodyConsumer from deploy call should not be null");
 
-    bodyConsumer.chunk(ChannelBuffers.wrappedBuffer(Bytes.toBytes(GSON.toJson(appRequest))), mockResponder);
+    byte[] contents = Bytes.toBytes(GSON.toJson(appRequest));
+    Preconditions.checkNotNull(contents);
```

Comment:
Should never happen. But ``Bytes.toBytes`` is ``@Nullable``, and Intellij doesn't like it. Maybe I should just add a ``@SuppressWarnings("ConstantConditions")``?